src/fields/ParagraphField.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+
+import React from 'react';
+import styled from 'styled-components';
+
+import { isNonEmptyString } from '../utils/LangUtils';
+
+const ParagraphWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex: 0 1 auto;
+`;
+
+const ParagraphTitle = styled.h2`
+  font-size: 16px;
+  font-weight: normal;
+  margin: 16px 0 0 0;
+`;
+
+const ParagraphBody = styled.div`
+  font-size: 14px;
+  margin: 12px 0 0 0;
+`;
```

Comment:
This should probably be up to the designers and is entirely cosmetic.

What about keeping the margins all in the `ParagraphTitle` and removing the margins in the body? I'm not expecting the margins to collapse, which means the effective distance between the Title and the body is 28px. Seems pretty large?
const ParagraphTitle = styled.div`
  ...
  margin: 16px 0;
`;

Reply:
Yes, it'd be nice to get input from our designers. However, I don't think designers should be concerned with implementation details, and I would argue that paddings and margins are considered implementation details.

I can go into details about why I chose these specific margins, but here's a visual of the difference between what I have and what you're suggesting:

As is:
<img width="1000" alt="Screen Shot 2020-01-28 at 4 03 56 PM" src="https://user-images.githubusercontent.com/440299/73316561-193f0500-41e8-11ea-9c8f-b7f93e320660.png">

Only `ParagraphTitle` has margins:
<img width="1000" alt="Screen Shot 2020-01-28 at 4 04 56 PM" src="https://user-images.githubusercontent.com/440299/73316562-19d79b80-41e8-11ea-8125-f0409928dd65.png">